src/pages/collab/rooms/[id].tsx

```diff
@@ -207,6 +211,61 @@ const Output = ({
   </div>
 );
 
+const Chatbox = ({

```

Comment:
I think its best if the communication related components be stored in the components folder?

Reply:
done

Reply:
done